UI/js/profile.js

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright (C) 2016
+Developed at University of Toronto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/**
+* Init function
+*/
+$(function () {
+    $('select').material_select();
+});
+
+$("#theme").on('change', function() {
+    $('bodyclass').removeClass();
```

Comment:
"->'
Var -> const for function definitions.
Define element as const at the beginning

Reply:
done